src/services/item/plugins/app/service.ts

```diff
@@ -32,6 +32,10 @@ export class AppService {
     return repositories.appRepository.getAll(publisherId);
   }
 
+  async getMostUsedApps(actor: Actor, repositories: Repositories, memberId: string) {
```

Comment:
You do not need to have a `memberId` parameter since you can get the current user of the request with the `actor` parameter. You can also change the type from `Actor` to `Member` since you know it should be defined.